bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSHandler.java

```diff
@@ -332,10 +333,34 @@ public void postUpdate(String channelId, State state) {
             updateState(channelId, state);
         }
 
-        // channel subscription only works when on livetv app.
-        if (CHANNEL_APP_LAUNCHER.equals(channelId) && APP_ID_LIVETV.equals(state.toString())) {
-            channelHandlers.get(CHANNEL_CHANNEL).refreshSubscription(CHANNEL_CHANNEL, this);
+        // channel subscription only works when livetv app is started,
+        // therefore we need to slightly delay the subscription
+        if (CHANNEL_APP_LAUNCHER.equals(channelId)) {
+            if (APP_ID_LIVETV.equals(state.toString())) {
+                scheduleChannelSubscriptionJob();
+            } else {
+                stopChannelSubscriptionJob();
+            }
+        }
+    }
+
+    private void scheduleChannelSubscriptionJob() {
+        ScheduledFuture<?> job = channelSubscriptionJob;
+        if (job == null || job.isCancelled()) {
+            logger.debug("Schedule channel subscription job");
+            channelSubscriptionJob = scheduler.schedule(
+                    () -> channelHandlers.get(CHANNEL_CHANNEL).refreshSubscription(CHANNEL_CHANNEL, this),
+                    CHANNEL_SUBSCRIPTION_DELAY_SECONDS, TimeUnit.SECONDS);
+        }
+    }
+
+    private void stopChannelSubscriptionJob() {
```

Comment:
You should call this during `dispose()` as well.

Reply:
very true